components/pages/explorer/getConfigError.tsx

```diff
@@ -22,22 +22,10 @@
 import { ReactNode } from 'react';
```

Comment:
This file customises the error message to remove references to Arranger Admin UI, etc.

WIP